### PR TITLE
[ADD] Frontdesk: moving visitors section to new doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Moving the 'Visitors' and 'Visitor Flow'  section to its own doc since it can stand alone- the main doc is too long. This is to hlp cut up the doc into more digestible parts. Updated icons and ref links.

[Project Card](https://www.odoo.com/odoo/project/3835/tasks/3946118?cids=3) for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
@brse-odoo - this is ready for a first round review!

**Comment by larm-odoo** [comment]
Thank you for your review, @Felicious! I implemented all the suggestions and added the seciton for check in thorugh check out.

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - this is ready for a final review!

**Review by ksc-odoo — APPROVED** [review]
hey, @larm-odoo -- just finished my Final Review. Nicely done! Approving now. Only have a handful of minor suggestions that require your attention. Once you implement the necessary changes, feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/hr/frontdesk/visitors.rst [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+When their drink has been served, click the :guilabel:`Drink Served` button to indicate the drink
+has been delivered to the guest. Once clicked, that button disappears.
+
+.. image:: visitors/visitors.png
+   :align: center
+   :alt: The full list of currently checked in visitors, with the drinks to be served highlighted.
+
+If any column is not visible, or if a visible column is preferred to be hidden, click the
+:icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, located at the end of the column name
```
[the remark this answers is not in the document]
Comment (larm-odoo) [reply content/applications/hr/frontdesk/visitors.rst:59]: Agreed- seems like we are still figurin out to use the UI terms that we see or the font awesome description. A perfetc discussion for a future writers workshop! I will also have @samueljlieber put in his 2 cents here when he does the tech rveiew.

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is ready for a tech review! And KSC and I had one comment we need you to weigh in on =)

File: content/applications/hr/frontdesk/visitors.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (samueljlieber) [reply content/applications/hr/frontdesk/visitors.rst]: Hi, to be consistent across all icon labels I think it is best to stick with the icon's name:
```rst 
:icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon...
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+All visitors are presented in a list view, with the following details that were entered upon check
+in:
+
+- :guilabel:`Name`: the guest's name.
+- :guilabel:`Visitor Company`: the company the guest represents.
+- :guilabel:`Phone`: the guest's phone number.
+- :guilabel:`Drinks*`: the drink the guest requested.
+- :guilabel:`Host`: who the guest is waiting to see.
+- :guilabel:`CheckIn`: the date and time the guest checked in.
+- :guilabel:`Checkout*`: the date and time the guest checked out. In the default view, only guests
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:30]:
```suggestion
- :guilabel:`Checkout`\*: the date and time the guest checked out. In the default view, only guests
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible. Guests with check-out
+  times are only visible if the :guilabel:`Today` filter is not active.
+- :guilabel:`Duration`: the amount of time the guest has been checked in for.
+- :guilabel:`Station`: the location of where the guest checked in.
+- :guilabel:`Status`: the status of the guest. The options are :guilabel:`Checked-In`,
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Canceled`.
+- :guilabel:`Email*`: the guest's email address.
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:37]:
```suggestion
- :guilabel:`Email`\*: the guest's email address.
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+\*\ These fields are not visible in the default :guilabel:`Visitor` list. These must be enabled
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:39]: Just need to escape before the formatting character :)
```suggestion
\* These fields are not visible in the default :guilabel:`Visitor` list. These must be enabled
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+using the :icon:`oi-settings-adjust` :guilabel:`(additional options)` icon in the top-right of the
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:40]: Please see [my comment](https://github.com/odoo/documentation/pull/9454#discussion_r1619445128) :)
```suggestion
using the :icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon in the top-right of the
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+.. image:: visitors/visitors.png
+   :align: center
+   :alt: The full list of currently checked in visitors, with the drinks to be served highlighted.
+
+If any column is not visible, or if a visible column is preferred to be hidden, click the
+:icon:`oi-settings-adjust` :guilabel:`(additional options)` icon, located at the end of the column
+name list.
+
+Clicking that reveals a drop-down menu of column options to enable or disable. A :icon:`fa-check`
+:guilabel:`(check)` icon indicates the column is visible.
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:68]: Update icon label, and refrain from "That" language, especially when starting a new paragraph. I was confused what you meant by "Clicking that". Suggesting to restructure to:
```suggestion
:icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, located at the end of the column
name list. Doing so, reveals a drop-down menu of column options to enable or disable. A
:icon:`fa-check` :guilabel:`(check)` icon indicates the column is visible.
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+Visitor flow
+============
+
+Visitor check in
+----------------
+
+When a visitor arrives at a facility, they approach a :ref:`Frontdesk kiosk <frontdesk/kiosk>`, and
+click :guilabel:`Check in`. The information requested from the visitor is what was configured for
+that specific *Frontdesk* station. If any information is required, the field displays a red asterisk
+(*). The visitor **must** enter the required information in order to check in.
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:105]:
```suggestion
(\*). The visitor **must** enter the required information in order to check in.
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+
+.. note::
+   At any point in the check-in process, if ten seconds pass with no selection, the kiosk returns to
+   the main welcome screen.
+
+Drinks
+------
+
+If drinks were configured for the station, after tapping :guilabel:`Check In`, a registration
+confirmation screen loads, along with the question: `Do you want something to drink?`
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:117]: Text that appears on the UI like this should be in a :guilabel: :)
```suggestion
confirmation screen loads, along with the question: :guilabel:`Do you want something to drink?`
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+
+The visitor can tap either :guilabel:`Yes, please`, or :guilabel:`No, thank you`.
+
+If they select :guilabel:`Yes, please`, a drink selection screen appears, and the pre-configured
+options are listed. The visitor then taps the desired selection, or, if they do not want anything,
+they can tap the :guilabel:`Nothing, thanks` button at the bottom of the screen.
+
+If a drink selection was made, a `Thank you for registering! Your drink is on the way.` message
+appears.
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:126]:
```suggestion
If a drink selection was made, a :guilabel:`Thank you for registering! Your drink is on the way.`
message appears.
```

File: content/applications/hr/frontdesk/visitors.rst [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+Notifications
+-------------
+
+Once the visitor has checked in, the person they are visiting, and any other users who were
+configured to be notified when check-ins occur at the kiosk, are notified. The notification is
+either by email, SMS message, a *Discuss* chat, or any combination of those three options.
+
+If the visitor requested a drink, the user(s) configured as the :guilabel:`People to Notify` on the
+drink form, are notified, via the *Discuss* application. The message that appears is: `(Visitor
+Name) just checked-in. They requested (Drink Name).`
```
Comment (samueljlieber) [inline content/applications/hr/frontdesk/visitors.rst:137]:
```suggestion
drink form, are notified, via the *Discuss* application. The message that appears is:
:guilabel:`(Visitor Name) just checked-in. They requested (Drink Name).`
```

**Pull request closed without merging**